net/wireguard/src/opnsense/mvc/app/controllers/OPNsense/Wireguard/forms/dialogEditWireguardServer.xml

```diff
@@ -57,7 +57,7 @@
         <style>tokenize</style>
         <type>select_multiple</type>
         <allownew>true</allownew>
-        <help>List of addresses to configure on the device. Please use CIDR notation like 10.0.0.1/24.</help>
+        <help>List of addresses to configure on the device. Please use CIDR notation like 10.0.0.1/24. Can be left empty for point to point connections.</help>
```

Comment:
I wouldn't mind leaving this out. The form will not complain and if the model ever changes the help text is wrong.

Reply:
Yes that makes sense I removed it.